src/pages/item/item.html

```diff
@@ -31,6 +31,6 @@
       </ion-item>
 		</ion-list>
 
-		<button ion-button (click)="onSave(itemForm)" type="submit" block>Save</button>
+		<button ion-button (click)="onSave(itemForm)" type="submit" block [disabled]="!selectedBrand || !selectedModel || !selectedCategory">Save</button>
```

Comment:
So it is not using HTML5 form validation, but I used a simple version of Angular form validation that I actually used in a few other pages and that is much cleaner.